flaskr/schema.sql

```diff
@@ -1,17 +1,20 @@
 DROP TABLE IF EXISTS user;
 DROP TABLE IF EXISTS post;
+DROP TABLE IF EXISTS task;
 
 CREATE TABLE user (
   id INTEGER PRIMARY KEY AUTOINCREMENT,
   username TEXT UNIQUE NOT NULL,
   password TEXT NOT NULL
 );
 
-CREATE TABLE post (
+CREATE TABLE task (
```

Comment:
We also need a `due_date` field. This will be the date on which the task shows up on UI.